Dynamic data structures/Marshalling yard/marchallingYard.c

```diff
@@ -0,0 +1,143 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include "stdio.h"
+#include "stdlib.h"
+#include "string.h"
```

Comment:
```suggestion
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
```
Файлы в кавычках препроцессор начинает искать с текущей директории. Он найдёт и так, но это будет дольше.